RadixWallet/Features/HomeFeature/Children/AccountRow/Home+AccountRow+Reducer.swift

```diff
@@ -47,20 +48,17 @@ extension Home {
 			switch viewAction {
 			case .task:
 				let accountAddress = state.account.address
-				if state.portfolio.wrappedValue == nil {
-					state.portfolio = .loading
-				}
-
 				self.checkAccountAccessToMnemonic(state: &state)
 
 				return .run { send in
-					for try await accountPortfolio in await accountPortfoliosClient.portfolioForAccount(accountAddress) {
+					for try await fiatWorth in await accountPortfoliosClient.portfolioForAccount(accountAddress).map(\.totalFiatWorth).removeDuplicates() {
```

Comment:
Update to observe just totalFiatWorth, thus reducing the number of potential updates